api.py

```diff
@@ -133,15 +133,19 @@ def wait_for_job(job_id, **kwargs):
     """
     while True:
         time.sleep(0.5)
-        response = make_get_request(job_id, **kwargs)
-        response.raise_for_status()
-
-        if response.ok:
-            msg = response.json()
-            if 'response' in msg:
+        try:
+            response = make_get_request(job_id, **kwargs)
+
+            if response.ok:
+                msg = response.json()
+                if 'response' in msg:
+                    return response
+        except requests.exceptions.HTTPError as e:
+            if e.response.status_code < 500:
```

Comment:
This will probably work in our case, since a stuck job will eventually time out and result in a 404, but should we be more robust here?  Perhaps breaking out after some reasonable length of time when we get nothing but 5XX errors?

Reply:
I was wondering that too, but to what end.  I am hoping this is good for now.

Considering a DHL ingest script. If we time out on 500s we'll just go to the next ingest job and start getting 500s there, presumably.